docs/docusaurus/docs/core/configure_project_settings/configure_metadata_stores/configure_metadata_stores.md

```diff
@@ -0,0 +1,118 @@
+   print(context.variables.stores['expectations_store'])
+   print(context.variables.stores['validation_definition_store'])
+   print(context.variables.stores['checkpoint_store'])
+   print(context.variables.stores['suite_parameter_store'])
+   print(context.variables.stores['validation_results_store'])
+   ```
+   
+   When updating a Store configuration you will use the same key to access it from your Data Context's `variables.stores` attribute as was used to print it in the above example.
+   
+3. Update the `base_path` of the Store to change.
+
+   Each Store has a `store_backend` configuration that determines how and where the Store accesses and saves information.  To change the location of a Store, you will update the `store_backend` configuration's `base_directory` value.  For instance, to change the location of an Expectation Store, you would update the `expectation_store_directory` variable in the following code and execute it:
+
+   ```pyhton title="Python"
+   expectation_store_directory = "my_expectations_store/"
+   context.variables.stores['expectations_store']['store_backend']['base_directory'] = expectation_store_directory
+   ```
+
+   The path provided for the `base_directory` should be either be an absolute path, or a path relative to the File Data Context's `project_root_dir`.
+
+4. Save the File Data Context variables.
+
+   Once a Data Context's `variables` have been updated, the changes need to be saved to the Data Context's configuration file so that they will persist when the Data Context is loaded in the future.  This is done with:
+
+   ```python title="Python"
+   context.variables.save()
+   ```
+
+5. Re-initialize the File Data Context.
+
+   Because Store configurations are loaded when the Data Context is initialized, you will need to re-initialize your Data Context before your changes will take effect.  This is done by loading the Data Context again, exactly as when it was loaded the first time:
+
+   ```python title="Python"
+   context = gx.get_context(mode="file")
+   ```
+
+</TabItem>
+
+<TabItem value="sample_code" label="Sample code">
+
+```python
+import great_expectations as gx
+
+# Load a File Data Context
+context = gx.get_context(mode="file")
+
+# Access the Stores through the Data Context's `variables` attribute:
+print(context.variables.stores['expectations_store'])
+print(context.variables.stores['validation_definition_store'])
+print(context.variables.stores['checkpoint_store'])
+print(context.variables.stores['suite_parameter_store'])
+print(context.variables.stores['validation_results_store'])
+
+# Update the path of the Data Context's Expectations Store:
+expectation_store_directory = "my_expectations_store/"
+context.variables.stores['expectations_store']['store_backend']['base_directory'] = expectation_store_directory
+
+# Save changes to the Data Context's configuration:
+context.variables.save()
+
+# Re-initialize the Data Context:
+context = gx.get_context(mode="file")
```

Comment:
I actually have to restart the ipython session. After updating the config as I describe earlier this always fails with:
```
context = gx.get_context(mode="file")
Invalid store configuration: Please check the configuration of your TupleFilesystemStoreBackend named expectations_store. Exception was: 
 'UUID' object has no attribute 'replace'
```

Reply:
This is actually quite an subtle bug. I've described it here: https://greatexpectations.atlassian.net/browse/V1-434

Reply:
I'll work on fixing this bug so we can merge the docs.

Reply:
This is new fixed and merged.